media/css/mozorg/internet-health.scss

```diff
@@ -0,0 +1,830 @@
+    h2 span {
+        background: $colorInclusion;
+    }
+
+    .content {
+        background-image: url('/media/img/mozorg/internet-health/inclusion.svg')
+    }
+
+    .footer-hr:after {
+        @include font-size(46px);
+        color: $colorInclusion;
+        content: '<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<';
+        letter-spacing: 0.2em;
+
+        @media #{$mq-tablet} {
+            @include font-size(60px);
+            line-height: 37px;
+        }
+
+        @media #{$mq-desktop} {
+            @include font-size(78px);
+            line-height: 48px;
+        }
+    }
+}
+
+#literacy {
+    h2 span {
+        background: $colorLiteracy;
+    }
+
+    .content {
+        background-image: url('/media/img/mozorg/internet-health/literacy.svg')
+    }
+
+    .footer-hr:after {
+        @include font-size(28px);
+        color: $colorLiteracy;
+        content: '@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@';
+        letter-spacing: 0.3em;
+
+        @media #{$mq-tablet} {
+            @include font-size(36px);
+            line-height: 45px;
+        }
+
+        @media #{$mq-desktop} {
+            @include font-size(48px);
+            line-height: 60px;
+        }
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Newsletter
+
+#newsletter-subscribe {
+    background: white;
+    color: black;
+    margin: 0;
+
+    a {
+        color: black;
+    }
+
+    button,
+    input[type='email'] {
+        border: 1px solid #0c99d5;
+    }
+
+    button {
+        background: #00e6ff;
+        color: #4a4a4a;
+    }
+
+    .content {
+        padding-top: 0;
+    }
+
+    .form-title {
+        background-image: url('/media/img/mozorg/internet-health/newsletter.svg');
+    }
+
+    @media #{$mq-tablet} {
+        padding: 40px 0;
+
+        .content {
+            padding-top: 20px;
+        }
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Blogs feed
+
+#blogs {
+    background: #f2f2f2;
+    padding: 60px 0;
+
+    header {
+        margin-bottom: 40px;
+
+        h2 {
+            @include font-size(36px);
+            font-weight: bold;
+        }
+
+        p {
+            @include font-size(16px);
+        }
+    }
+
+    ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+
+        li {
+            margin: 0;
+            padding: 0;
+        }
+    }
+
+    a:link,
+    a:visited {
+        text-decoration: none;
+    }
+
+    a:hover,
+    a:active,
+    a:focus {
+        text-decoration: underline;
+    }
+
+    article {
+        margin-bottom: 20px;
+        padding-bottom: 40px;
+        position: relative;
+
+        h3 {
+            @include font-size(20px);
+            font-weight: bold;
+            line-height: 1.3;
+            max-width: 24em;
+
+            a:link,
+            a:visited {
+                color: $colorLink;
+            }
+
+            a:hover {
```

Comment:
Actually are these nested link selectors needed? There seems to be a set further above on line 160 that you could use to style the link color.